server/src/main/java/org/candlepin/model/Owner.java

```diff
@@ -397,4 +418,48 @@ public void setLogLevel(String logLevel) {
     public String getName() {
         return getDisplayName();
     }
+
+    /**
+     * Utility method that defaults to 'entitlement'.
+     *  This covers legacy owners
+     *
+     * @return access mode.
+     */
+    @XmlTransient
+    public String contentAccessMode() {
+        return getContentAccessMode() == null ? DEFAULT_CONTENT_ACCESS_MODE : getContentAccessMode();
+    }
+
+    /**
+     * Returns the value of the contentAccessMode setting.
+     *
+     * @return String the value
+     */
+    public String getContentAccessMode() {
+        return contentAccessMode;
+    }
+
+    public void setContentAccessMode(String contentAccessMode) {
+        this.contentAccessMode = contentAccessMode;
+    }
+
+    /**
+     * Returns the value of the contentAccessModeList setting.
+     *
+     * @return String the value
+     */
+    public String getContentAccessModeList() {
+        return contentAccessModeList;
+    }
+
+    public void setContentAccessModeList(String contentAccessModeList) {
+        this.contentAccessModeList = contentAccessModeList;
+    }
+
+    public String[] getContentAccessModeListAsArray() {
```

Comment:
Need a @JsonIgnore annotation here so this doesn't show up in the serialized output.